### PR TITLE
feat:  Add FCR 2 Metal Redundant Connection Example

### PR DESCRIPTION
**Pull request by d-bhola**
- The PR contains a new example for FCR 2 Metal Redundant Connection. 
- This includes one FCR, one Metal Network and two connections - Primary and Secondary (Redundant) 
- The changes are tested in prod and redundant connection is successfully created.

### CODE REVIEW

**Comment by d-bhola** [comment]
Addressed all PR comments 👍

**Review by srushti-patl — APPROVED** [review]
LGTM

File: examples/cloud-router-2-metal-nimf-redundant-connection/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+This example shows how to leverage the [Fabric Cloud Router Connection Module](../../modules/cloud-router-connection/README.md)
+to create a Fabric Connection from a Fabric Cloud Router to Equinix Metal with two connections. A primary and secondary connection.
+
+It leverages the Equinix Terraform Provider, and the Fabric Cloud Router Connection
+Module to setup the connection based on the parameters you have provided to this example; or based on the pattern
+you see used in this example it will allow you to create a more specific use case for your own needs.
+
+See example usage below for details on how to use this example.
+
+<!-- Begin Example Usage (Do not edit contents) -->
```
Comment (thogarty) [inline examples/cloud-router-2-metal-nimf-redundant-connection/README.md:12]: These don't belong. We're not leveraging them anymore. Please remove.

File: examples/cloud-router-2-metal-nimf-redundant-connection/terraform.tfvars.example [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+notifications_type               = "ALL"
+notifications_emails             = ["example@equinix.com","test1@equinix.com"]
+purchase_order_number            = "1-323292"
+bandwidth                        = "50"
+project_id                       = "<Fabric_Project_ID>"
+zside_ap_type                    = "METAL_NETWORK"
+primary_direct_rp_name           = "pri_direct_rp"
+secondary_direct_rp_name         = "sec_direct_rp"
+primary_direct_equinix_ipv4_ip   = "190.1.1.25/30"
+secondary_direct_equinix_ipv4_ip = "190.1.1.9/30"
```
Comment (thogarty) [inline examples/cloud-router-2-metal-nimf-redundant-connection/terraform.tfvars.example:24]: I would reorder the example variables to match the ordering present in variables.tf files.

**Review by thogarty — CHANGES_REQUESTED** [review]
Changes requested.

File: examples/cloud-router-2-metal-nimf-redundant-connection/README.md (hunk 1 above)
Comment (thogarty) [reply examples/cloud-router-2-metal-nimf-redundant-connection/README.md:12]: Clarity on comment: we don't need these boundary declarations anymore `<!-- Begin Example Usage (Do not edit contents) -->`. They were for the previous shell script methods but now that we just use terraform-docs then we won't have any need for these. Hope that clears it up.

Comment (d-bhola) [reply examples/cloud-router-2-metal-nimf-redundant-connection/README.md]: Addressed with [commit](74747cd2cf07ab77431eb1de6c1f69e1749e578a)

File: examples/cloud-router-2-metal-nimf-redundant-connection/terraform.tfvars.example (hunk 2 above)
Comment (d-bhola) [reply examples/cloud-router-2-metal-nimf-redundant-connection/terraform.tfvars.example:27]: Addressed with [commit](74747cd2cf07ab77431eb1de6c1f69e1749e578a)

**Review by thogarty — APPROVED** [review]
LGTM

**Pull request merged**